react-native/react/menubar/index.render.desktop.js

```diff
@@ -1,50 +1,78 @@
 /* @flow */
+/*eslint-disable react/prop-types */ // Since we're using flow types for props
 
 import React, {Component} from '../base-react'
 import {FlatButton} from 'material-ui'
+import resolveAssets from '../../../desktop/resolve-assets'
 
-import {ipcRenderer} from 'electron'
+import commonStyles from '../styles/common'
 
-import {remote} from 'electron'
-
-import commonStyles, {colors} from '../styles/common'
+const publicFolder = `file:///${resolveAssets('../react-native/react/images/folders/fa-folder-public-empty.svg')}`
+const privateFolder = `file:///${resolveAssets('../react-native/react/images/folders/fa-folder-private-empty.svg')}`
 
 export default class Render extends Component {
   props: {
+    username: ?string,
+    openingMessage: ?string,
+    openKBFS: () => void,
+    openKBFSPublic: () => void,
+    openKBFSPrivate: () => void,
+    showMain: () => void,
+    showHelp: () => void,
+    quit: () => void,
+    openingButtonInfo: {
+      text: string,
+      onClick: () => void
+    },
     debug?: boolean
   };
 
   render (): ReactElement {
+    const {openKBFS, openKBFSPublic, openKBFSPrivate, showMain, showHelp, quit, openingButtonInfo} = this.props
+
     return (
-      <div>
-        <Header/>
-        <OpeningMessage/>
-        <FolderList/>
-        <Footer debug={this.props.debug || false}/>
+      <div style={{backgroundColor: '#f6f6f4', width: 320}}>
```

Comment:
we shoudl discuss the styling. we shouldn't have hardcoded colors etc in here. we have colors injected into the commonStyles